templates/isdialogmote/avlysning-arbeidsgiver.hbs

```diff
@@ -0,0 +1 @@
+{{> isdialogmote/partials/base items=this doctitle="Dialogmøte" heading="Avlysning av dialogmøte" brevtitle="Brev til nærmeste leder"}}
```

Comment:
Kunne denne informasjonen heller blitt sendt med i json og så lagt til derifra, f.eks som en HEADING? Da hadde denne appen fortsatt å være lite kompleksitet her og man hadde ikke trengt noen nye filer eller endepunkter.

Reply:
Ja, tror det du foreslår kan funke Erik. Men kanskje det beste da er å definere BREVTITLE som en egen type (dvs documentcomponenttype). Da trenger man bare utvide base.hbs med 
   {{#eq type 'BREVTITLE' }}
        <h1 class="centered">>
            {{#each texts }}
                {{ this }}<br/>
            {{/each}}
        </h1>
    {{else eq type 'HEADER' }}...
    
    Da blir endringene ellers bare i Modia-koden (må legge til de nye brev-overskriftene der), mens i isdialogmote blir det ikke annet enn å utvide DocumentComponentType med BREVTITLE.

Reply:
Jeg prøvde dette opprinnelig, men jeg klarte ikke helt å finne ut hvordan det blir i .hbs filen. Men jeg kan godt titte litt mer på det.